standalone/src/models/quote/quote-request/constants.ts

```diff
@@ -13,7 +13,7 @@ export const TAX_MODE = {
   Disabled: 'Disabled',
 } as const;
 
-export const TAX_ROUNDING_MODE = {
+export const ROUNDING_MODE = {
```

Comment:
Why do we need to change this name?
The constants are exported in the models so this would break consumers which are using this one.